src/components/calcite-shell-panel/calcite-shell-panel.tsx

```diff
@@ -24,6 +24,16 @@ export class CalciteShellPanel {
    */
   @Prop({ reflect: true }) collapsed = false;
 
+  /**
+   * floatContent lets the content area appear like a floating panel and only sets a max-height.
+   */
+  @Prop({ reflect: true }) floatContent = false;
+
+  /**
+   * floatContentScale sets the max-height of the content area when floatContent is true.
+   */
+  @Prop({ reflect: true }) floatContentScale: "s" | "m" | "l" = "m";
```

Comment:
Since we'll probably use `"s" | "m" | "l"` quite frequently, maybe we should have a type somewhere for `CalciteScale`?

```ts
type CalciteScale = "s" | "m" | "l";
```

Reply:
I like it!

Reply:
Like maybe in *interfaces.ts*?

Reply:
yep!

Reply:
Added a separate issue for this.

Reply:
#593